sqlx-cli/src/opt.rs

```diff
@@ -62,6 +62,10 @@ pub enum DatabaseCommand {
 /// Group of commands for creating and running migrations.
 #[derive(Clap, Debug)]
 pub struct MigrateOpt {
+    /// Path to folder containing migrations. Defaults to 'migrations'
+    #[clap(long)]
+    pub source: Option<String>,
```

Comment:
You could use `default_value = "migrations"` instead of `Option<String>`

Reply:
@abonander much better, thanks for the tip!